src/packages/excalidraw/README.md

```diff
@@ -2,6 +2,10 @@
 
 Excalidraw exported as a component to directly embed in your projects.
 
+#### Note
+
+You are viewing the docs for the **next** release, in case you want to check the docs for the stable release, you can view it [here](https://www.npmjs.com/package/@excalidraw/excalidraw).
```

Comment:
I'd add some ⚠⚠⚠ emojis here :)

Reply:
also maybe remove this on build when publishing so it's not in the stable readme?

Reply:
I am removing this in build already :
<img width="374" alt="Screenshot 2022-07-04 at 8 47 32 PM" src="https://user-images.githubusercontent.com/11256141/177182596-c0ce3a87-24b0-4cdb-be8d-b975ce7bc4be.png">
)